src/AbstractToggleWidget.php

```diff
@@ -11,7 +11,7 @@
 
 abstract class AbstractToggleWidget extends Widget
 {
-    protected array $toggleOptions = [];
+    protected array $toggleOptions = array();
```

Comment:
Why? It should be `[]`.

Reply:
Sorry, this is my experiment with Rector. I'll fix it.